ios/RNFIRMesssaging.m

```diff
@@ -240,10 +240,9 @@ - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMess
   [_bridge.eventDispatcher sendDeviceEventWithName:FCMNotificationReceived body:[remoteMessage appData]];
 }
 
-RCT_EXPORT_METHOD(presentLocalNotification:(UNNotificationRequest *)data resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+RCT_EXPORT_METHOD(presentLocalNotification:(UNNotificationRequest *)request resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
@muszu Thank you for your help.
You changed data into request but please check line 254.
data exists! ^^;
I changed this line and tested.
Crash was resolved but I couldn't confirm any notifications!
Please check this?